code/modules/surgery/bodyparts/parts.dm

```diff
@@ -70,30 +70,41 @@
 	bodytype = BODYTYPE_LARVA_PLACEHOLDER | BODYTYPE_ORGANIC
 	acceptable_bodytype = BODYTYPE_LARVA_PLACEHOLDER
 
-/obj/item/bodypart/l_arm
+/obj/item/bodypart/arm
```

Comment:
if you're going to add an abstract parent type, make it explicitly clear in every possible way that this item should not exist in game.